<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgDeviceAjust</class>
 <widget class="QDialog" name="DlgDeviceAjust">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>1280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pb2">
   <property name="geometry">
    <rect>
     <x>481</x>
     <y>202</y>
     <width>157</width>
     <height>157</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pbInfo">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>1160</y>
     <width>115</width>
     <height>120</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pbHome">
   <property name="geometry">
    <rect>
     <x>718</x>
     <y>0</y>
     <width>82</width>
     <height>82</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pbTitle">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>274</width>
     <height>82</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labTime">
   <property name="geometry">
    <rect>
     <x>563</x>
     <y>1179</y>
     <width>237</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labDate">
   <property name="geometry">
    <rect>
     <x>563</x>
     <y>1225</y>
     <width>237</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
